libraries/AP_HAL_ChibiOS/hwdef/VUAV-V7pro/README.md

```diff
@@ -0,0 +1,607 @@
+   <td>GND</td>
+   </tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### POWER2
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN_H</td>
+   <td>+24V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>CAN_L</td>
+   <td>+24V</td>
+   </tr>
+   <td>5</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+## RC Input
+
+RC input is configured on the RCIN pin, at one end of the servo rail. This pin supports all unidirectional RC protocols.
+
+## PWM Output
+
+The VUAV-V7pro supports up to 14 PWM outputs,support all PWM protocols as well as DShot. All 14 PWM outputs have GND on the top row, 5V on the middle row and signal on the bottom row.
+
+The 14 PWM outputs are in 4 groups:
+
+ - PWM 1, 2, 3 and 4 in group1
+ - PWM 5, 6, 7 and 8 in group2
+ - PWM 9, 10, 11 and 12 in group3
+ - PWM 13, 14 in group4
+
+Channels 1-8 support bi-directional Dshot.
+Channels within the same group need to use the same output rate. If any channel in a group uses DShot, then all channels in that group need to use DShot.
+
+## GPIOs
+
+All 14 PWM channels can be used for GPIO functions (relays, buttons, RPM etc).
+The pin numbers for these PWM channels in ArduPilot are shown below:
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>PWM Channels</th>
+   <th>Pin</th>
+   <th>PWM Channels</th>
+   <th>Pin</th>
+   </tr>
+   <tr>
+   <td>PWM1</td>
+   <td>50</td>
+   <td>PWM8</td>
+   <td>57</td>
+   </tr>
+   <tr>
+   <td>PWM2</td>
+   <td>51</td>
+   <td>PWM9</td>
+   <td>58</td>
+   </tr>
+   <tr>
+   <td>PWM3</td>
+   <td>52</td>
+   <td>PWM10</td>
+   <td>59</td>
+   </tr>
+   <tr>
+   <td>PWM4</td>
+   <td>53</td>
+   <td>PWM11</td>
+   <td>60</td>
+   </tr>
+   <tr>
+   <td>PWM5</td>
+   <td>54</td>
+   <td>PWM12</td>
+   <td>61</td>
+   </tr>
+   <tr>
+   <td>PWM6</td>
+   <td>55</td>
+   <td>PWM13</td>
+   <td>62</td>
+   </tr>
+   <tr>
+   <td>PWM7</td>
+   <td>56</td>
+   <td>PWM14</td>
+   <td>63</td>
+   </tr>
+   </tbody>
+   </table>
+
+## Analog inputs
+
+The VUAV-V7pro flight controller has 5 Analog inputs
+
+ - ADC Pin2-> Battery Current
+ - ADC Pin16 -> Battery Voltage 
+ - ADC Pin19 -> ADC 3V3 Sense
+ - ADC Pin3 -> ADC 6V6 Sense
+ - ADC Pin9 -> RSSI voltage monitoring
+
+## Battery Monitor Configuration
+
+The board has voltage and current inputs sensor on the POWER1 ADC and POWER2 CAN connector.
+
+The correct battery setting parameters are:
+
+Enable Battery1 monitor:
+ - BATT_MONITOR   4
+ - BATT_VOLT_MULT 15.7 (may need adjustment if supplied monitor is not used)
+ - BATT_AMP_PERVLT 60.61 (may need adjustment if supplied monitor is not used)
+

```

Comment:
```suggestion

- BATT_VOLT_PIN 16
- BATT_CUR_PIN 2
```

Reply:
I previously considered that the default PIN would be this value, so I removed these two parameters here. I am now adding them again.